sdk/src/main/java/com/mapbox/maps/renderer/MapboxRenderThread.kt

```diff
@@ -420,9 +419,9 @@ internal class MapboxRenderThread : Choreographer.FrameCallback {
     if (renderThreadPrepared && !paused) {
       draw()
     }
+    awaitingNextVsync = false
```

Comment:
Since order of calls is important here - lets add detailed comments explaining it